### PR TITLE
Abilities refactor.

### PR DESCRIPTION
**Pull request by MoMannn**
- assignAbilities -> grantAbilities
- Using constants.
- Updates description.

### CODE REVIEW

File: packages/0xcert-ethereum-utils-contracts/src/contracts/permission/abilitable.sol [hunk 1]
```diff
@@ -7,14 +7,14 @@ import "../math/safe-math.sol";
...
+ * 00000001 Ability A - number representation 1
+ * 00000010 Ability B - number representation 2
+ * 00000100 Ability C - number representation 4
+ * 00001000 Ability D - number representation 8
+ * 00010000 Ability E - number representation 16
  * etc ... 
- * To assign abilities 2 and 3 we would need a bitfield of 00000110 which is represented by number 6
- * in other words the sum od ability 2 and 3. The same concept works for revoking abilities and 
+ * To grant abilities B and C we would need a bitfield of 00000110 which is represented by number 6
+ * in other words the sum od ability B and C. The same concept works for revoking abilities and
```
Comment (typwrtr) [inline packages/0xcert-ethereum-utils-contracts/src/contracts/permission/abilitable.sol]: `To grant abilities B and C, we would need a bitfield of 00000110 which is represented by number 6, in other words, the sum of abilities B and C. The same concept works for revoking abilities and checking if someone has multiple abilities.`

File: packages/0xcert-ethereum-xcert-contracts/src/core/types.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export enum XcertAbilities {
+  MANAGE_ABILITIES = 1,
+  CREATE_ASSET = 2,
+  REVOKE_ASSET = 4,
+  TOOGLE_TRANSFERS = 8,
```
Comment (typwrtr) [inline packages/0xcert-ethereum-xcert-contracts/src/core/types.ts]: `TOGGLE_TRANSFERS = 8`

File: packages/0xcert-ethereum-xcert-contracts/src/tests/xcert.test.ts [hunk 3]
```diff
@@ -122,24 +123,24 @@ spec.test('throws when trying to create an xcert to zero address', async (ctx) =
...
-  ctx.not(logs.events.AssignAbilities, undefined);
+  const logs =  await xcert.instance.methods.grantAbilities(bob, XcertAbilities.CREATE_ASSET).send({ from: owner });
+  ctx.not(logs.events.GrantAbilities, undefined);
 
   const bobHasAbility1 = await xcert.instance.methods.isAble(bob, 2).call();
   ctx.is(bobHasAbility1, true);
 });
 
-spec.test('throws when a third party tries to assign create ability', async (ctx) => {
+spec.test('throws when a third party tries to grants create ability', async (ctx) => {
```
Comment (typwrtr) [inline packages/0xcert-ethereum-xcert-contracts/src/tests/xcert.test.ts]: `spec.test('throws when a third party tries to grant create ability', async (ctx) => {`

File: packages/0xcert-ethereum-asset-ledger/src/core/ledger.ts [hunk 4]
```diff
@@ -170,11 +170,11 @@ export class AssetLedger implements AssetLedgerBase {
   }
 
   /**
-   * Gives an account abilities.
+   * Grants an account abilities.
```
Comment (typwrtr) [inline packages/0xcert-ethereum-asset-ledger/src/core/ledger.ts]: `Grants abilities of an account.`
